notebooks/ranking_signals.ipynb

```diff
@@ -0,0 +1,1005 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Objective Ranking Signals Quickstart\n",
+    "\n",
+    "Objective’s AI-native search ranks search results by a relevance score. Ranking Signals enable you to adjust the score, and therefore the final ranking of search results at query-time through a new query parameter.\n",
+    "\n",
```

Comment:
Should we mention the name of the parameter? Like, "search results at query-time with the new `ranking_expr` parameter."